### ISSUE TITLE
🐛 [firebase_vertexai] Add output schema just like google_generative_ai

### ISSUE THREAD

**Issue by imhafeez** [firebase/flutterfire#13209]
## What feature would you like to see?

Is there a missing a feature that is supported on an underlying SDK? Or is there a feature that you think would be useful to have in FlutterFire?

In google_generative_ai you can control the output schema of your prompt. `GenerationConfig.responseSchema` but its missing in firebase_vertexai.

**Comment by MaggieThomann** [comment]
Yes please!! Big +1 to this. Seems like it would be simple enough to add.

**Comment by cynthiajoan** [comment]
Added in GA PR https://github.com/firebase/flutterfire/pull/13453
